app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -6,7 +6,9 @@ def create_in_provider(manager_id, params)
       params = provider_params(params) if respond_to?(:provider_params)
       manager = ExtManagementSystem.find(manager_id)
       tower_object = provider_collection(manager).create!(params)
-
+      if respond_to?(:refresh_in_provider)
```

Comment:
dont want to be the spoilsport, but `refresh_new_target` is for the old refresh and this is using the graph refresh.
But good news follows: afaik the graph refresh supports that out of the box. [ManagerRefresh::Target](https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/target.rb) 
@Ladas can you point to a small code sample in amazon, where you create objects from hashes?